Makefile.core.mk

```diff
@@ -459,6 +461,9 @@ operator-test:
 .PHONY: mixer-test
 mixer-test: mixer-racetest
 
+.PHONE: featuresgen-test
+featuresgen-test: featuresgen-racetest
```

Comment:
why do we need a whole new test target just for this one file?

Reply:
It seemed like everything else, such as mixer, had it's own test target, so I followed along. What test should I make it a part of?

Reply:
Oh I didn't realize this was a whole new top level folder. This should be under pkg/test, which already has a target

Reply:
Ah I must have misunderstood where Mitch wanted me to put these files, I'll move them to that folder